framework/src/components/logger/logger.js

```diff
@@ -61,6 +61,7 @@ const createLogger = ({ fileLogLevel, consoleLogLevel, logFileName }) => {
 		name: 'lisk-framework',
 		streams,
 		src: consoleSrc || fileSrc,
+		serializers: bunyan.stdSerializers,
```

Comment:
What if instead of using all serialisers we only use `bunyan.stdSerializers.err` ?

Reply:
Good one @pablitovicente